packages/bpk-mixins/src/mixins/_margins.scss

```diff
@@ -19,7 +19,7 @@
 /// RTL aware leading margin
 ///
 /// @param {string} $margin The desired leading margin.
-/// @param {boolean} $resetOppositeMargin [false] Whether to reset the opposite margin to 0.
+/// @param {boolean} $resetOppositeMargin [true] Whether to reset the opposite margin to 0.
```

Comment:
Corrected the docs here as the default value state was incorrect

Reply:
We should probably mention this in the changelog